scripts/install.tpl.sh

```diff
@@ -33,10 +33,20 @@ function kubectl_wait() {
     kubectl wait --for=condition=Available --namespace="${namespace}" "${runtime}" --timeout="${timeout}"
 }
 
+function kubectl_wait_rollout() {
+    namespace=$1
+    runtime=$2
+    timeout=$3
+
+    kubectl rollout status --namespace="${namespace}" "${runtime}" --timeout="${timeout}"
+}
+
 kubectl apply -f "https://github.com/cert-manager/cert-manager/releases/download/${cert_mgr_version}/cert-manager.yaml"
 kubectl_wait "cert-manager" "deployment/cert-manager-webhook" "60s"
 
 kubectl apply -f "https://github.com/operator-framework/catalogd/releases/download/${catalogd_version}/catalogd.yaml"
+# Wait for the rollout, and then wait for the deployment to be Available
+kubectl_wait_rollout "olmv1-system" "deployment/catalogd-controller-manager" "60s"
```

Comment:
The deployment is updated with the latest cert secret, so this will wait for the rollout to complete, and the next step will ensure that it's ready.